spec/formatting.md

```diff
@@ -942,9 +942,9 @@ The _Default Bidi Strategy_ is defined as follows:
       True if the `u:dir` _option_ of the _resolved value_ of `exp` has a value other than `'inherit'`,
       or False otherwise.
    1. If `dir` is `'LTR'`:
-      1. If `msgdir` is `'LTR'` in the formatted output
+      1. If `msgdir` is `'LTR'`
          and `isolate` is False,
-         let `fmt` be itself
+         let the formatted output be `fmt` itself.
```

Comment:
This change implies replacing _all_ of the formatted output with `fmt`, not just replacing `exp` with the contents of `fmt`. So I would not make this change.

Reply:
Ah, yes. Fixed in [39e9056](https://github.com/unicode-org/message-format-wg/pull/969/commits/39e9056592d0aa4bfe8da978ee646f0f6737e751)